### ISSUE TITLE
Requirements - suggestions

### ISSUE THREAD

**Issue by pawelqs** [drugilsberg/chimaera#1]
I have just set up the environment and it was not easy.

1) pandas was unavaiable for python 3.4
2) using numpy 1.15 with hdbscan results in the following error: https://github.com/scikit-learn-contrib/hdbscan/issues/272 so one needs to upgrade numpy.

I am running chimaera with python 3.6.0 and numpy 1.17.2. Requirements' update may save the other users' time.

Best!

**Comment by drugilsberg** [comment]
HI @pawel125 thanks a lot I will update the requirements. Will close as soon is done.

**Comment by drugilsberg** [comment]
Closed by #2 and #3.

**Comment by brucemoran** [comment]
Just to let you know, on trying to install using `https` method:
```
Due to incompatibilities with Python 3.7 hdbscan now requires Cython to be installed
```
Which resolved fine after `pip install Cython`

**Comment by drugilsberg** [comment]
Thanks @brucemoran, will update the setup instructions to reflect this.
Closing the issue as soon as the instructions are up-to-date.

**Comment by drugilsberg** [comment]
Implemented the changes and closing in #5
